src/System.Linq.Queryable/ref/System.Linq.Queryable.cs

```diff
@@ -67,6 +67,8 @@ public static partial class Queryable
         public static float Average<TSource>(this System.Linq.IQueryable<TSource> source, System.Linq.Expressions.Expression<System.Func<TSource, float>> selector) { throw null; }
         public static System.Linq.IQueryable<TResult> Cast<TResult>(this System.Linq.IQueryable source) { throw null; }
         public static System.Linq.IQueryable<TSource> Concat<TSource>(this System.Linq.IQueryable<TSource> source1, System.Collections.Generic.IEnumerable<TSource> source2) { throw null; }
+        public static System.Linq.IQueryable<TSource> Append<TSource>(this System.Linq.IQueryable<TSource> source, TSource element) { throw null; }
```

Comment:
ref file should have entries sorted (or use the tool that builds them, though I had some problem trying to install it and just worked by hand when I first hit a ref file).